src/pymorize/cmorizer.py

```diff
@@ -126,6 +127,8 @@ def __init__(
         self._post_init_populate_rules_with_dimensionless_unit_mappings()
         self._post_init_populate_rules_with_aux_files()
         self._post_init_populate_rules_with_data_request_variables()
+        self._post_init_create_controlled_vocabularies()
+        #self._post_init_populate_global_attributes()
```

Comment:
Why is this commented out?

Reply:
Because the global attributes are set in the pipeline, not as a `_post_init` job, right @siligam?

Reply:
sorted!